### PR TITLE
Add Error tab to host expense pipeline tabs

### PR DESCRIPTION
**Pull request by gustavlrsn**
Adding an "Error" tab to the Host Expense pipeline preview feature:

image

### CODE REVIEW

File: components/host-dashboard/HostDashboardExpenses.js [hunk 1]
```diff
@@ -214,6 +217,12 @@ const initViews = [
     showCount: true,
     id: 'incomplete',
   },
+  {
+    label: 'Error',
```
Comment (Betree) [inline components/host-dashboard/HostDashboardExpenses.js:221]: We should internationalize these tabs

File: components/host-dashboard/HostDashboardExpenses.js [hunk 2]
```diff
@@ -123,6 +123,9 @@ const hostDashboardMetaDataQuery = gql`
       @include(if: $getViewCounts) {
       totalCount
     }
+    error: expenses(host: { slug: $hostSlug }, limit: 0, offset: 0, status: ERROR) @include(if: $getViewCounts) {
```
Comment (Betree) [inline components/host-dashboard/HostDashboardExpenses.js:126]: Woot? I didn't know our Apollo Server already supports `@include` directives; that's a game-changer! Would be worth sharing on slack, I don't think anyone in the engineering team is aware of that

**Review by Betree — APPROVED** [review]

File: components/host-dashboard/HostDashboardExpenses.js (hunk 2 above)
Comment (gustavlrsn) [reply components/host-dashboard/HostDashboardExpenses.js:126]: @Betree Had to double-check it actually works now, and thankfully yes, the server does not resolve those fields when the value is false

File: components/host-dashboard/HostDashboardExpenses.js (hunk 1 above)
Comment (gustavlrsn) [reply components/host-dashboard/HostDashboardExpenses.js]: Fixed this

File: components/host-dashboard/HostDashboardExpenses.js (hunk 2 above)
Comment (Betree) [reply components/host-dashboard/HostDashboardExpenses.js:126]: Cool! Let's let the others know in `#engineering` :-)

**Pull request merged**